packages/wayland.rb

```diff
@@ -29,10 +29,11 @@ def self.build
       set +a
     EOF
     
-    system "meson #{CREW_MESON_FNO_LTO_OPTIONS} \
-
-    -Ddocumentation=false \
-    builddir"
+    system <<~CONFIGURE
+      meson #{CREW_MESON_FNO_LTO_OPTIONS} \
+      -Ddocumentation=false \
+      builddir
+    CONFIGURE
```

Comment:
What exactly is fixed here?  This CONFIGURE string is not referenced anywhere else in the package.

Reply:
I removed the empty line in `line 33`

Reply:
Why use `<<~CONFIGURE` instead of just using double quotes?

Reply:
I guess one advantage would be you don't have to escape double quotes, if needed.